vllm/v1/worker/tpu_model_runner.py

```diff
@@ -1499,3 +1567,25 @@ def _get_padded_token_len(paddings: list[int], x: int) -> int:
     index = bisect.bisect_left(paddings, x)
     assert index < len(paddings)
     return paddings[index]
+
+
+def _get_padded_lora_rank(max_lora_rank: int, max_num_loras: int) -> int:
+    max_num_loras += 1
```

Comment:
why is there a +1?

Reply:
So vllm puts in a dummy lora with just 0s for tokens that aren't using any lora adapters, but the dummy doesn't count towards the lora count